Delete user

jho44 opened the pull request:
enable deleting user

lilith commented:
But if one adult creates a household, adds another adult, and that adult deletes their account then the household is deleted too. Thus we planned to have a flow where you don't delete the household unless you're the last adult

jho44 commented:
> But if one adult creates a household, adds another adult, and that adult deletes their account then the household is deleted too. Thus we planned to have a flow where you don't delete the household unless you're the last adult

Ah yes...that is perfectly reasonable. Just added that logic and changed the wording of the confirmation modal
